mis_builder/models/mis_report_instance.py

```diff
@@ -500,14 +504,19 @@ def _compute_query_company_ids(self):
     @api.model
     def get_filter_descriptions_from_context(self):
         filters = self.env.context.get('mis_report_filters', {})
-        analytic_account = filters.get('analytic_account_id', {})
-        analytic_account_id = analytic_account.get('value')
+        analytic_account_id = filters.get('analytic_account_id', {}).get('value')
         filter_descriptions = []
         if analytic_account_id:
             analytic_account = self.env['account.analytic.account'].browse(
                 analytic_account_id)
             filter_descriptions.append(
                 _("Analytic Account: %s") % analytic_account.display_name)
+        analytic_tag_id = filters.get('analytic_tag_ids', {}).get('value')
```

Comment:
s/analytic_tag_ids/analytic_tag_id/ ?

Reply:
Yes because the filter is a m2o (select one tag at a time) but we are looking on the field "analytic_tag_ids" on move.line. Therefore the name of the filter is the name of the field on which we will search.